nexus/db-model/src/schema_versions.rs

```diff
@@ -29,6 +29,7 @@ static KNOWN_VERSIONS: Lazy<Vec<KnownVersion>> = Lazy::new(|| {
         // |  leaving the first copy as an example for the next person.
         // v
         // KnownVersion::new(next_int, "unique-dirname-with-the-sql-files"),
+        KnownVersion::new(125, "bp-physical-disk-state"),
```

Comment:
Probably need a different name for this migration (and definitely need different `upX.sql` implementations)

Reply:
For sure. Done in e2e2cd5016481ca8d89349f361afba78d8d1af07